.github/workflows/centos7-ci.yml

```diff
@@ -29,6 +29,15 @@ jobs:
       with:
         submodules: recursive
 
+    - name: Build rpm package
+      run: |
+        sudo gem install --no-document fpm
+        # WOULD CHANGE TO MASTER WHEN PR GETS MERGED
+        git clone https://github.com/idbeta/build-tools.git
```

Comment:
IMO, we should wait for `https://github.com/api7/apisix-build-tools/pull/14` to be merged, and use `api7/apisix-build-tools`

Reply:
Thanks, it's only for test for now.